OKR.md

```diff
@@ -2,6 +2,34 @@
 
 We try to frame our ongoing work using a process based on quarterly Objectives and Key Results (OKRs). Objectives reflect outcomes that are challenging, but realistic. Results are tangible and measurable.
 
+## 2018 Q4
+
+# A viable alternative to Go IPFS
+
+* P0 - @vasco-santos - DHT is enabled by default and interoperates with Go IPFS DHT
+* P1 - @vasco-santos - IPNS is functional over pubsub and the DHT
+* P0 - @alanshaw - Base32 CIDv1 migration is complete
+* P0 - @alanshaw - A JS IPFS daemon is part of the IPFS gateways
+* P2 - @hugomrdias - JS IPFS runs on the same ports as Go IPFS and uses the same binary name
+* P2 - @hugomrdias - An IPFS version manager CLI tool exists to switch between go/js/whatever versions e.g. nvm/nave
+* P0 - @achingbrain - npm-on-ipfs is the default registry that IPFS developers and CI use to install npm dependencies
+
+# Measurably faster and smaller than ever before
+
+* P0 - @hugomrdias - JS IPFS is capable of transferring a large dataset like ImageNet at least 75% as fast as Go IPFS
+* P2 - @hugomrdias - The browser bundle of IPFS is not more than 1.5MB (gzipped) in size
+* P3 - @hugomrdias - Browser bundle size is measured over time and developers alerted if it increases above 1.5MB
+* P1 - @alanshaw - A website is published to visually track a set of basic performance profiles of JS IPFS releases over historical versions
+
+# Stable and future proof
+
+* P0 - @hugomrdias - Error codes are added to all errors that originate directly from JS IPFS
+* P2 - @alanshaw - Test quality is improved, duplicates removed and coverage increased
+* P0 - @achingbrain - js-ipfs-unixfs-engine is decomposed into smaller modules for better maintainability, testability and reliability. The refactored modules are part of a JS IPFS release
```

Comment:
Make sure to extract the HAMT so that it can be used as a generic KV store on top of IPLD //cc @mikeal